applications/openshift/kubelet/var_kubelet_evictionhard_imagefs_inodesfree.var

```diff
@@ -11,7 +11,7 @@ operator: equals
 interactive: false
 
 options:
-  default: "5%"
+  default: "10%"
```

Comment:
Why was this value changed? In the PR, you said that the kubelet default is 5% compiled in (I don't see the value reported trough the proxy object) and with the PR, this check is failing. (btw it is failing also with the current upstream checks).

Reply:
I have reverted this one back, since we are going to drop the no present eviction rules from all the profiles